src/main/scala/scalafix/sbt/ScalafixEnable.scala

```diff
@@ -61,73 +55,14 @@ object ScalafixEnable {
         extracted,
         maybeSemanticdbScalacFullScalaVersion
       )
-      enableSemanticdbPlugin <- maybeFullVersion.toList.flatMap { fullVersion =>
-        List(
-          scalaVersion := fullVersion,
-          SemanticdbPlugin.semanticdbVersion := BuildInfo.scalametaVersion
-        )
-      } :+ (SemanticdbPlugin.semanticdbEnabled := true)
+      enableSemanticdbPlugin <- maybeFullVersion.toList.map { fullVersion =>
+        scalaVersion := fullVersion,
+      } :+ (semanticdbEnabled := true)
       settings <-
         inScope(ThisScope.copy(project = Select(p)))(
           scalacOptionsSettings ++ enableSemanticdbPlugin
-        )
+        ) :+ (Global / semanticdbVersion := BuildInfo.scalametaVersion)
```

Comment:
As explained in the commit, this makes `scalafixEnable` work properly on sbt 1.3.[0-3]. This is also broken on master since https://github.com/scalacenter/sbt-scalafix/commit/5e14eb5f4471f960954050454fda54bac0b11d62, but using 1.3.0 for scripted tests exposed this.